purchase_request/README.rst

```diff
@@ -0,0 +1,153 @@
+you have to choose a supplier.
+
+In case that you chose to select an existing RFQ, the application will search
+for existing lines matching the request line, and will add the extra
+quantity to them, recalculating the minimum order quantity,
+if it exists for the supplier of that RFQ.
+
+In case that you create a new RFQ, the request lines will also be
+consolidated into as few as possible lines in the RFQ.
+
+**Table of contents**
+
+.. contents::
+   :local:
+
+Configuration
+=============
+
+To configure the product follow this steps:
+
+#. Go to a product form.
+#. Go to *Inventory* tab.
+#. Check the box *Purchase Request* along with the route *Buy*.
+
+With this configuration, whenever a procurement order is created and the supply
+rule selected is 'Buy' the application will create a Purchase Request instead
+of a Purchase Order.
+
+Usage
+=====
+
+Purchase requests are accessible through a new menu entry 'Purchase
+Requests', and also from the 'Purchase' menu.
+
+Users can access the list of Purchase Requests or Purchase Request Lines.
+
+It is possible to filter requests by its approval status.
+
+Bug Tracker
+===========
+
+Bugs are tracked on `GitHub Issues <https://github.com/OCA/purchase-workflow/issues>`_.
+In case of trouble, please check there if your issue has already been reported.
+If you spotted it first, help us smashing it by providing a detailed and welcomed
+`feedback <https://github.com/OCA/purchase-workflow/issues/new?body=module:%20purchase_request%0Aversion:%2014.0%0A%0A**Steps%20to%20reproduce**%0A-%20...%0A%0A**Current%20behavior**%0A%0A**Expected%20behavior**>`_.
+
+Do not contact contributors directly about support or help with technical issues.
+
+Credits
+=======
+
+Authors
+~~~~~~~
+
+* ForgeFlow
+
+Contributors
+~~~~~~~~~~~~
+
+* Jordi Ballester Alomar <jordi.ballester@forgeflow.com>
+* Jonathan Nemry <jonathan.nemry@acsone.eu>
+* Aaron Henriquez <ahenriquez@forgeflow.com>
+* Adrien Peiffer <adrien.peiffer@acsone.eu>
+* Lois Rilo <lois.rilo@forgeflow.com>
+* Héctor Villarreal <hector.villarreal@forgeflow.com>
+* Ben Cai <ben.cai@elico-corp.com>
+* Rattapong Chokmasermkul <rattapongc@ecosoft.co.th>
+
+Other credits
+~~~~~~~~~~~~~
+
+The development of this module has been financially supported by:
+
+.. image:: https://upload.wikimedia.org/wikipedia/en/3/3b/Aleph_Objects_Logo.png
+   :alt: Aleph Objects, Inc
+   :target: https://www.alephobjects.com
+
+Images
+------
+
+* Enric Tobella (logo)
+
+Maintainers
+~~~~~~~~~~~
+
+This module is maintained by the OCA.
+
+.. image:: https://odoo-community.org/logo.png
+   :alt: Odoo Community Association
+   :target: https://odoo-community.org
+
+OCA, or the Odoo Community Association, is a nonprofit organization whose
+mission is to support the collaborative development of Odoo features and
+promote its widespread use.
+
+This module is part of the `OCA/purchase-workflow <https://github.com/OCA/purchase-workflow/tree/14.0/purchase_request>`_ project on GitHub.
```

Comment:
please replace 14.0 by 15.0

Reply:
README will be regenerated after merge is done